docs(wallet button): explain new feature wallet button for woocommerce (#1220)
mercadopago/devsite-docs: guides/plugins/woocommerce/preferences.en.md

| Configuration | Description |
| --- | --- |
| Discount coupon | Activate this option when you want to offer discounts to your customers. A field will appear on the form where they can enter their coupon. |

> | Payment with stored cards | Payment with card stored in Mercado Pago function, for Mercado Pago customers to buy without having to fill in card data at the store's checkout. |

Reviewer: ```suggestion
| Payment with stored cards | Allows customers to use their card data saved in Mercado Pago without having to fill in card data at the store's checkout. |
```
